### PR TITLE
first commit of the censorship blog

### PR DESCRIPTION
**Pull request by deborahtrez**
Blog about Contributing to Mixxx amid Uganda's internet censorship directives.

@ywwg  @Holzhaus kindly review :)

### CODE REVIEW

File: content/news/2021-01-24-internet-censorship.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
+title: "About my Outreachy internship with Mixxx DJ: Contributing to open source amidst Uganda’s internet restrictions"
+authors: Aanyu Deborah Oduman
+tags: personal, outreachy, internship
+date: 2021-01-24 14:00:00
+summary: Last week, amid the 2021 general election, Uganda was disconnected from the internet entirely. The country experienced a widespread internet blackout that lasted 4 days, starting on the eve of the election (13th January 2021) and ending in the morning of 18th January 2021. In the days leading up to the election, access to major social media platforms and circumvention tools was blocked – even when the OTT (Over the Top) tax (commonly referred to as the “Social Media Tax”) was paid. This internet shutdown and the social media ban that followed affected many online jobs and businesses, my internship with Mixxx under the Outreachy program was no exception.
```
Comment (ywwg) [inline content/news/2021-01-24-internet-censorship.md]: this should be the first paragraph of the article, not the summary.  The summary could be something like:

"In the days leading up to the Ugandan election, access to major social media platforms and circumvention tools was blocked. This internet shutdown and the social media ban that followed affected many online jobs and businesses, my internship with Mixxx under the Outreachy program was no exception."

File: content/news/2021-01-24-internet-censorship.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+By the time it dawned on us that the internet really had been disconnected, we were already in darkness, with only the people physically close to us to express our shock too.
+The audacity of this government!
+
+As the years go by, our President, H.E Yoweri Kaguta Museveni, has been feeling a lot of pressure about the presidential seat.
+If elections were conducted freely and fairly, you would notice a declining trend in the number of votes his political party (NRM) has been getting over the years.
+Ugandans are tired, and they want change. They have been more vocal about the injustices that have been happening in the country and it has cost them their lives, even their families’.
+This resistance from the public has been causing anxiety in State House, and it’s starting to show.
+
+In the 2016 presidential elections, Ugandan authorities blocked access to Facebook, Twitter and WhatsApp saying the platforms would be used by the opposition to mobilize protests.
+(See [[this article from the BBC](https://www.bbc.com/news/world-africa-35601220))
```
Comment (ywwg) [inline content/news/2021-01-24-internet-censorship.md]: Maybe rephrase to "This article from the bbc has more context on the social media blockage"

File: content/news/2021-01-24-internet-censorship.md [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+President Yoweri Museveni was facing a tough challenge from veteran activist [Kizza Besigye](https://twitter.com/kizzabesigye1?s=20), and he believed shutting down social media would help to curb the “threat”.
+Authorities also suspended mobile transfers of money.
+
+In 2019, the government introduced a tax on the use of the social media, which activists have called an attempt at controlling free speech.
+He said that the youth only use social media to gossip and spread wrong rumors (about him).
+(See [this article from the BBC](https://www.bbc.com/news/world-africa-44315675)) .
```
Comment (ywwg) [inline content/news/2021-01-24-internet-censorship.md]: Again, give a hint about what the article contains so the reader can decide if they want to click the link.

File: content/news/2021-01-24-internet-censorship.md [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+He imposed OTT – Over the Top Tax and since then, it’s never been enough to simply have a good internet connection.
+You have to pay a daily fee of UGX 200 to your Internet Service Provider, or if you’re being rebellious, a VPN application installed on your phone or computer.
+He insisted that the revenue collected from the tax would help the country cope with the consequences of olugambo [gossiping].
+I don’t know.. but to me, these look like the actions of an increasingly anxious person in power.
```
Comment (ywwg) [inline content/news/2021-01-24-internet-censorship.md]: should be "I don't know..." (three dots, not two)

File: content/news/2021-01-24-internet-censorship.md [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+
+On 12th January 2021, the Uganda Communications Commission (UCC) ordered internet service providers in the country to immediately suspend any access and use of all messaging apps and social media platforms until further notice.
+This was done so suddenly and without prior notice that it took most of us a couple of failed OTT subscriptions to realize that social media platforms had indeed been blocked.
+Before that, internet speeds had become excruciatingly slow, it was any wonder we were able to get anything done in those few days.
+A few days after the social media block, the government decided to shut down the entire internet! (The social media ban was simply not cutting it)
+So when Google stopped working and our parents were restarting their phones several times, is when we realized that truly, there was no end to the surprises.
+
+During this [not so brief] period we were in total blackout. No internet services were available anywhere.
```
Comment (ywwg) [inline content/news/2021-01-24-internet-censorship.md]: use () parentheses for asides, not [] brackets.

**Review by ywwg — APPROVED** [review]
This is really good, and I have only a few notes

File: content/news/2021-01-24-internet-censorship.md (hunk 1 above)
Comment (deborahtrez) [reply content/news/2021-01-24-internet-censorship.md]: Thank you

File: content/news/2021-01-24-internet-censorship.md (hunk 5 above)
Comment (deborahtrez) [reply content/news/2021-01-24-internet-censorship.md]: Alright

**Review by ywwg — APPROVED** [review]
lgtm!

**Comment by ywwg** [comment]
I will merge if you have no more changes you want to make

**Comment by deborahtrez** [comment]
> I will merge if you have no more changes you want to make

I don't think I will be making any more changes :) You can go ahead and merge it.
